### PR TITLE
6.1.5 release notes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_6.1.md [hunk 1]
```diff
@@ -49,6 +49,99 @@ This issue will be corrected in an upcoming release.
...
+
+### Resolved Issues
+
+- **The following CVEs have been resolved in this release:** I95-51431, I95-51758, I95-52495, I95-52496, I95-52497, I95-52509, I95-52554, I95-52625, I95-52645, I95-52956, 
+
+- **I95-48174 Expand supported values for DHCP option:** DHCP option 43 is now a supported option, as well as a binary encoded-type (hex/byte) support. Valid examples are `0xabcdef` and `0x123456`.
+------
+- **I95-50671 Office365 traffic is not recognized by SVR Routers:** Resolved an issue where application module traffic was being miscategorized and therefore not fully qualified. Module traffic ingressing over SVR is no longer miscategorized as an internet service when the module is present on both sides.
+------
+- **I95-50708 Significant decrease in memory usage for the salt_master process:** **NEED INFO FROM DEV ON THIS - STILL WAITING**
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.1.md]: After reviewing the defect notes, here is my take:

- **I95-50708 Time series data for memory of the salt_master process periodically significantly decreases:** Incorrect method for polling application memory data; this resulted in dips in application memory being presented.

File: docs/release_notes_128t_6.1.md [hunk 2]
```diff
@@ -49,6 +49,99 @@ This issue will be corrected in an upcoming release.
...
+------
+- **I95-51336 App-ID stats entry not getting cleaned up after expiration:** In some cases, a session may not have installed correctly, preventing the expired App-ID stats from being removed. The App-ID stats entries are now cleaned up appropriately.
+------
+- **I95-51450 Support for 100/Full Speed/Duplex on Intel I225-V Driver NICs:** The DPDK driver has been updated to allow fixed speed and duplex configuration to work with IGC i225 NICs.
+------
+- **I95-51492 Password expiration not working:** This issue has been resolved. Adminstrators must use the global setting `configure authority password-policy lifetime N ` to indicate that all user passwords must be changed every `N` days.
+------
+- **I95-51638 Traceroute does not complete over SSR, but does using linux:** The traceroute command was unable to resolve through some network elements. The default 128T traceroute UDP port number has been changed to a more common/recognized port.
+------
+- **I95-51766 Enable TX lockup detector for bond interfaces:** The datapath lockup detection mechanism has been re-enabled to run on bond interfaces.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.1.md:84]: Suggested rewording:

- **I95-51766 TX lockup detector not enabled for LAG/bonded interfaces:** The datapath lockup detection mechanism has been re-enabled to run on bond interfaces.

File: docs/release_notes_128t_6.1.md [hunk 3]
```diff
@@ -49,6 +49,99 @@ This issue will be corrected in an upcoming release.
...
+------
+- **I95-51992 Multi-queue support for Bond interfaces:** Support for a bond `device-interface` to use multiple RX/TX queues has been added. 
+------
+- **I95-52104 URI escape characters handled incorrectly:** The `lookup application by-domain` and `clear app-id cache-entry url` were handling url parameters incorrectly, in lookup, creating and clearing cache entries. This has been resolved and each command now performs the correct operation.
+------
+- **I95 52105 Permissions error when attempting to `delete certificate webserver`:** Resolved an issue where `delete certificate webserver` and `create cerificate webserver` with an existing certificate were failing. On older versions of software this can be worked around by running `sudo rm -rf /etc/128technology/pki/webserver.pem`.
+------
+- **I95-52113 Application Director consumes a full core:** Resolved an overrun bug that was causing the Application Director to enter an endless loop when loading port ranges. This issue has been resolved.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.1.md:96]: Suggested rewording:

- **I95-52113 Application Identification on the SSR runs at 100% CPU utilization:** Resolved an overrun bug that was causing the SSR to enter a loop when loading port ranges. This issue has been resolved.

File: docs/release_notes_128t_6.1.md [hunk 4]
```diff
@@ -49,6 +49,99 @@ This issue will be corrected in an upcoming release.
...
+------
+- **I95-52305 Compacting rate limit exceeded:** Resolved memory and CPU issues resulting from attempting to compact very large application identification documents.
+------
+- **I95-52402 Router stuck in `Upgrading` state:** Resolved an issue with `conductor-only` mode, where the conductor was attempting to download the installer before the software access proxies were in place, preventing an update to the installer.
+------
+- **I95-52480 Conductor shows alarms when applications are added to the router configuration:** A condition has been added that verifies whether the node is a router or conductor before running application update and generating alarms on a conductor.
+------
+- **I95-52491 Crash in `metadata propagate egress` with indications that the PacketControlBlock was corrupted:** Resolved an issue when metadata is segmented across two packet buffers, a function used in `metadata propagate egress` discarded the packet buffer which housed all of the relevant PacketControlBlock data. the segmented packets are no longer discarded and the dataplane no longer crashes when processing a packet comprised of segmented metadata.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.1.md:115]: Suggested rewording:
- **I95-52491 Crash in highway process due to segmented metadata** Resolved an issue processing metadata that is segmented across two packet buffers. The segmented packets are no longer discarded and the dataplane no longer crashes when processing a packet comprised of segmented metadata.

File: docs/release_notes_128t_6.1.md [hunk 5]
```diff
@@ -49,6 +49,99 @@ This issue will be corrected in an upcoming release.
...
+------
+- **I95-52968 Configuration template not being applied by the conductor to a router during upgrade:** Resolved an issue where a managed router would silently fail to apply a configuration from a conductor with a greater software version if the configuration contained features that the router's earlier SSR version did not understand. Specifically, if a config field such as an enumeration, that the router was aware of, had new values added to it in the conductor's version of software then the configuration would not be applied.
+------
+- **I95-52971 Inconsistent hash and signing of RPM files:** Some small number of RPM files did not usen the `sha256` hash for sigining. This has been corrected and all RPM packages on the distribution ISO are digest sha256 for Common Criteria.
+------
+- **WAN-2090 Conductor managed SSR Applications in WAN Insights Showing up as Numbers:** Resolved an issue with stats APIs, which were not properly handling some internal service names.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.1.md:135]: Showing should not be capitalized

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_6.1.md [hunk 6]
```diff
@@ -49,6 +49,99 @@ This issue will be corrected in an upcoming release.
...
+
+### Caveats
+
+- **I95-52426 Incorrect behavior when configuring an IDP custom rule definition:** In a case where a user is modifying a rule to **decrease** the action type to an `alert`, alerts for that vulnerability will not work. The attack will be allowed to pass through undetected. For example, if the action `close-tcp-connection` is downgraded to `alert`, the attacks will pass through undetected.
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_6.1.md]: The rule generation issue was addressed. There still is an issue with alerts, per James Silvia.

File: docs/bcp_service-policy_defaults.md [hunk 7]
```diff
@@ -3,7 +3,7 @@ title: Service Policy Baseline Configurations
 sidebar_label: Service Policy Baseline Defaults
 ---
 
-One of the most powerful aspects of the SSR data model is the flexibility offered by a `service-policy`. Giving customers the ability to configure thresholds for when traffic should migrate from path to path based on current network status is a daunting task, however; most customers – and even application developers – are unaware of the sensitivity of their service to latency, loss, and jitter.
+ne of the most powerful aspects of the SSR data model is the flexibility offered by a `service-policy`. Giving customers the ability to configure thresholds for when traffic should migrate from path to path based on current network status is a daunting task, however; most customers – and even application developers – are unaware of the sensitivity of their service to latency, loss, and jitter.
```
Comment (MichaelBaj) [inline docs/bcp_service-policy_defaults.md:6]: Accidentally removed the 'O'

File: docs/config_command_guide.md [hunk 8]
```diff
@@ -8,7 +8,7 @@ sidebar_label: Configuration Command Guide
 
 ## `configure authority`
 
-Authority configuration is the top-most level in the SSR configuration hierarchy.
+Authority configuration is the top-most level in the 128T router configuration hierarchy.
```
Comment (MichaelBaj) [inline docs/config_command_guide.md:11]: This should read SSR configuration, and all instances throughout

**Review by MichaelBaj — DISMISSED** [review]

File: docs/release_notes_128t_6.1.md [hunk 9]
```diff
@@ -49,6 +49,97 @@ This issue will be corrected in an upcoming release.
...
+  - `best-match-only` considers the best matching prefix length. In cases of transport overlap, services are visited in alphabetical order.
+  - Using `any-match` will consider all services that match the route update but do not have the best match service address when creating FIB entries, minimizing missed entries. The transports from the service with the longest prefix are considered first.
+------
+- **I95-52517 Allow users the ability to configure the OSPF SPF timers:** Support for user-configured values for SPF delay has been added. Users can now specify values for spf delay, hold-time, and maximum-hold-time. For additional information, see [OSPF SPF Timers](config_command_guide.md#configure-authority-router-routing-ospf-timers-spf).
+
+### Resolved Issues
+
+- **The following CVEs have been resolved in this release:** I95-51431, I95-51758, I95-52495, I95-52496, I95-52497, I95-52509, I95-52554, I95-52625, I95-52645, I95-52956.
+
+- **I95-48174 Expand supported values for DHCP option:** DHCP option 43 is now a supported option, as well as a binary encoded-type (hex/byte) support. Valid examples are `0xabcdef` and `0x123456`.
```
Comment (hadrielk) [inline docs/release_notes_128t_6.1.md:70]: this is arguably a feature rather than a bug fix.

**Review by jamesesilvia — DISMISSED** [review]

**Review by MichaelBaj — DISMISSED** [review]

**Review by avinashbhat80 — APPROVED** [review]

File: docs/release_notes_128t_6.1.md [hunk 10]
```diff
@@ -49,6 +49,96 @@ This issue will be corrected in an upcoming release.
...
+------
+- **I95-52113 Application Identification on the SSR runs at 100% CPU utilization:** Resolved an overrun bug that was causing the SSR to enter a loop when loading port ranges. This issue has been resolved.
+------
+- **I95-52158 Spoke is rejecting hub BFD packets, and peering is unable to come up over LTE:** In a corner case where the spoke private LTE IP changes before BFD is up and the public/hub-received IP stays the same, the hub gets stuck in the init state. 
+This issue has been resolved; the LTE IP change is now handled it as a source-nat change, where the flows and actions can be recreated with the updated LTE private IP.
+------
+- **I95-52208 Metrics queries return incomplete data when FIPS is enabled:** Resolved an issue where a FIPS-incompatible hashing function was causing missing or incomplete metrics data. 
+------
+- **I95-52279 Prevent VRRP Virtual MACs from being dropped by Bond interface in LACP mode:**  Packets with VRRP virtual dest MACs are now processed by the Bond interface when using LACP.
```
Comment (ayungelson) [inline docs/release_notes_128t_6.1.md:103]: I think we should remove this as this was only a partial fix and didn't work for x710 NICs. The final fix for VRRP should come in 6.1.6.

File: docs/release_notes_128t_6.1.md [hunk 11]
```diff
@@ -49,6 +49,99 @@ This issue will be corrected in an upcoming release.
...
 
+## Release 6.1.5
+
+**Release Date:** September 8, 2023
+
+### New Features
+
+- **I95-52163 Handle incoming public keys from peer conductor node:** Added functionality to allow conductor nodes to share the authorized keys of managed routers between each other. If the SSH public key is retrieved from a managed router by one conductor node, then it is automatically shared with its conductor peer node.
+------
+- **I95-52316 Enhancements to Overlapping FIB Services:** The [`fib-service-match`](config_command_guide.md#configure-authority-fib-service-match) command allows you to configure either `best-match-only` or `any-match`.
```
[the remark this answers is not in the document]
Comment (jeffmart-jnpr) [reply docs/release_notes_128t_6.1.md:62]: @MichaelBaj is this issue resolved?

File: docs/release_notes_128t_6.1.md [hunk 12]
```diff
@@ -49,6 +49,99 @@ This issue will be corrected in an upcoming release.
...
+------
+- **I95-52316 Enhancements to Overlapping FIB Services:** The [`fib-service-match`](config_command_guide.md#configure-authority-fib-service-match) command allows you to configure either `best-match-only` or `any-match`. 
+  - `best-match-only` considers the best matching prefix length. In cases of transport overlap, services are visited in alphabetical order.
+  - Using `any-match` will consider all services that match the route update but do not have the best match service address when creating FIB entries, minimizing missed entries. The transports from the service with the longest prefix are considered first.
+------
+- **I95-52517 Allow users the ability to configure the OSPF SPF timers:** Support for user-configured values for SPF delay has been added. Users can now specify values for spf delay, hold-time, and maximum-hold-time. For additional information, see [OSPF SPF Timers](config_command_guide.md#configure-authority-router-routing-ospf-timers-spf).
+
+### Resolved Issues
+
+- **The following CVEs have been resolved in this release:** I95-51431, I95-51758, I95-52495, I95-52496, I95-52497, I95-52509, I95-52554, I95-52625, I95-52645, I95-52956,
```
[the remark this answers is not in the document]
Comment (jeffmart-jnpr) [reply docs/release_notes_128t_6.1.md]: has been updated

File: docs/release_notes_128t_6.1.md (hunk 1 above)
Comment (jeffmart-jnpr) [reply docs/release_notes_128t_6.1.md]: Looks like this has been updated.

File: docs/release_notes_128t_6.1.md [hunk 13]
```diff
@@ -49,6 +49,99 @@ This issue will be corrected in an upcoming release.
...
+------
+- **I95-52158 Spoke is rejecting hub BFD packets, and peering is unable to come up over LTE:** In a corner case where the spoke private LTE IP changes before BFD is up and the public/hub-received IP stays the same, the hub gets stuck in the init state. 
+This issue has been resolved; the LTE IP change is now handled it as a source-nat change, where the flows and actions can be recreated with the updated LTE private IP.
+------
+- **I95-52208 Metrics queries return incomplete data when FIPS is enabled:** Resolved an issue where a FIPS-incompatible hashing function was causing missing or incomplete metrics data. 
+------
+- **I95-52279 Prevent VRRP Virtual MACs from being dropped by Bond PMD in LACP mode:**  Packets with VRRP virtual dest MACs are now processed by the Bond PMD when using LACP.
```
[the remark this answers is not in the document]
Comment (jeffmart-jnpr) [reply docs/release_notes_128t_6.1.md]: Looks like this is resolved

File: docs/release_notes_128t_6.1.md (hunk 4 above)
Comment (jeffmart-jnpr) [reply docs/release_notes_128t_6.1.md]: Looks to be resolved

**Review by MichaelBaj — APPROVED** [review]

**Review by jeffmart-jnpr — APPROVED** [review]

**Pull request merged**